index.html

```diff
@@ -117,6 +117,194 @@ <h3 class="ingredients-header section-heading">Fine ingredients</h3>
           </div>
         </figure>
       </section>
+
+      <section class="menu">
+        <ul class="meal-group-list">
+          <li>
+            <article class="meal-group">
+              <h3 class="section-heading menu-header">Appetisers</h3>
+              <img
+                src="/img/divider.png"
+                class="divider"
+                alt="Decorated divider"
+              />
+              <ul class="meal-list">
+                <li>
+                  <dl class="meal">
+                    <div class="meal-line">
+                      <dt class="title">Tzatsiki</dt>
+                      <dt class="meal-price">$3.99</dt>
+                    </div>
+                    <dd class="section-text meal-text">
+                      Refreshing traditional cucumber and garlic yoghurt dip.
+                    </dd>
+                  </dl>
+                </li>
+                <li>
+                  <dl class="meal">
+                    <div class="meal-line">
+                      <dt class="meal-name">Aubergine Salad</dt>
+                      <dt class="meal-price">$5.50</dt>
```

Comment:
The price is description details
```suggestion
                      <dd class="meal-price">$5.50</dt>
```